content/kubermatic/v2.16/concepts/kubermaticconfiguration/_index.en.md

```diff
@@ -14,5 +14,5 @@ The following is an example configuration, showing all possible options. Note th
 don't define explicitly are always defaulted to these values.
 
 ```yaml
-{{< readfile "kubermatic/master/data/kubermaticConfiguration.yaml" >}}
+{{< readfile "kubermatic/v2.16/data/kubermaticConfiguration.yaml" >}}
```

Comment:
There is even better way to handle this in simple cases that is future proof.

Copy kubermatic/v2.16/data/kubermaticConfiguration.yaml right next to this file, and change diff to:

```suggestion
{{< readfile "kubermaticConfiguration.yaml" >}}
```

That will ensure future uses (made by recursive copying).

Reply:
According to the docs of Hugo the  filename must be relative to the current project working directory, or the project’s /content folder.